.gitignore

```diff
@@ -154,7 +154,7 @@ designer_version.h
 *.sw?
```

Comment:
This patch solves the following issues:

1. For iOS `build_version.hpp` wasn't generated at all because Xcode doesn't use cmake
2. For Android `build_version::kName` didn't contain flavorName and buildType.

This old/new implementation from 1y old patch fixes both problems by adding two methods to Platform. The version code a.k.a. intVersion has been reduced to yy_MM_dd. It matches dataVersion (also int32) and it should be more than enough for any conditionals in the code.